src/test/java/ru/ayaz/VideoTest.java

```diff
@@ -0,0 +1,55 @@
+package ru.ayaz;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+public class VideoTest {
+
+    private Video video;
+
+    @Before
+    public final void before() {
+        this.video = new Video("FunnyCats", mock(File.class));
```

Comment:
Agree. Should i configure video and other objects inside **@Before** when it take 3 or more rows in order to avoid code duplication?
&#x1F34F; **Resolved**